skymp5-server/cpp/server_guest_lib/script_classes/PapyrusActor.cpp

```diff
@@ -330,6 +330,11 @@ VarValue PapyrusActor::AddToFaction(VarValue self,
     resultFaction.rank = 0;
 
     actor->AddToFaction(resultFaction);
+
+    auto serializedArgs = SpSnippetFunctionGen::SerializeArguments(arguments);
```

Comment:
should remove on all clients (all neighbors). see PapyrusObjectReference::Disable